draft-dekater-scion-pki.md

```diff
@@ -1303,21 +1344,63 @@ The steps required to create a new AS certificate are the following:
 
 # Security Considerations
 
-The entire document is about security considerations.
-More details will follow in future versions of this draft.
+This section describes the possible security risks and attacks that SCION's control-plane PKI may be prone to, and how these may be mitigated. As SCION is an inter-domain network architecture, the focus lies on *inter*-AS trust issues. All *intra*-AS trust- and security aspects are out of scope.
+
+**Note:** This section only discusses security considerations related to SCION's control-plane PKI. For SCION control plane- and routing-specific security considerations, see {{I-D.scion-cp}}. {{I-D.scion-dp}} includes security considerations that concern the SCION data plane and data forwarding.
+
+
+## Kill Switches
+
+In a public key infrastructure, the certificate authorities have both the responsibility and power to issue and revoke certificates. If a CA is compromised or malicious, this power can unavoidably be abused. A misbehaving CA can plainly refuse to issue certificates to legitimate entities, or even issue illegitimate certificates to allow impersonating another entity. In the context of the SCION control plane PKI, refusing an AS to obtain or renew their AS certificate will ultimately cut the AS off from the network, turning the CP-PKI into a network "kill switch".
+
+SCION’s trust architecture fundamentally differs from a global monopolistic trust model. In SCION, each ISD manages its own trust roots instead of a single global entity providing those roots. This structure gives each ISD autonomy in terms of key management and in terms of trust. This prevents SCION from the occurrence of a global kill switch affecting all ISDs at once. However, local ISD kill switches are still possible in SCION. The following sections explain these cases and possible countermeasures.
+
+
+### Local ISD Kill Switch
+
+Compared to DNSSEC and BGPsec, a parent authority cannot switch off SCION core ASes, as they manage their own cryptographic trust roots. But executing a kill switch *inside* a local ISD can be done at different levels of the AS-level hierarchy:
+
+- On TRC level: The private root keys of the root certificates contained in an TRC are used to sign the CA certificates. If one of these private root keys is compromised, the adversary could issue illegitimate CA certificates, which may be used in further attacks. To maliciously change the entire TRC through a TRC update is more complicated: Each TRC update requires multiple different voting keys. The number of comprimised voting keys needed to perform a malicious update of a TRC depends on the voting quorum set in the TRC. The higher the quorum, the more complicated maliciously updating a TRC will be.
```

Comment:
Right? I'm not fan of the wording here. As far as I understand, compromise of a root certificate means compromise of one of the CAs. Maybe we could reword the paragraph to mention this? 

Also, in principle a CA has  both a root certificate and a CA certificate (according to [here](https://docs.scion.org/en/latest/cryptography/certificates.html#cp-root-certificate), right? Are there cases where a CA certificate is issued to a different organization?

Reply:
It would be good to mention that the CA certificates are not revoked, but short lived.

Reply:
> It would be good to mention that the CA certificates are not revoked, but short lived.


Done in https://github.com/scionassociation/scion-cppki_I-D/pull/25/commits/41bc0a15cb19372761910aa2fa620f7b27cccbdc